spec/models/hotel_spec.rb

```diff
@@ -4,4 +4,27 @@
   describe 'relationships' do
     it {should have_many :rooms}
   end
+
+  it '#name' do
+    budapest = Hotel.create!(name: "Grand Budapest", location: 'Turkey')
+    expect(budapest.name).to eq("Grand Budapest")
+  end
+
+  it '#location' do
+    budapest = Hotel.create!(name: "Grand Budapest", location: 'Turkey')
+    expect(budapest.location).to eq("Turkey")
+  end
+
+  it '#guests' do
+    budapest = Hotel.create!(name: "Grand Budapest", location: 'Turkey')
+    room1 = Room.create!(hotel_id: budapest.id, suite: "Presidential", rate: 125)
+    charlize = Guest.create!(name:"Charlize Theron", nights: 3)
+    danny = Guest.create!(name:"Danny", nights: 2)
+    mark = Guest.create!(name:"Mark", nights: 3) 
+    guest_room1 = GuestRoom.create!(room_id: room1.id, guest_id: charlize.id)
+    guest_room2 = GuestRoom.create!(room_id: room1.id, guest_id: danny.id)
+    guest_room3 = GuestRoom.create!(room_id: room1.id, guest_id: mark.id)
+    guest_room4 = GuestRoom.create!(room_id: room1.id, guest_id: charlize.id)
+    expect(budapest.guests.sort).to eq([charlize, danny, mark].sort)
+  end
 end
```

Comment:
You do not need to test for attributes on your Models. We can safely assume that ActiveRecord is making those available for us.

Great job on your `#guests` test. I'm wondering why you have to sort both arrays. Would something like `.to contain_exactly()` made it so you wouldn't have to sort?

Reply:
Thanks for that, Abdul. I just forgot how to compare two unsorted arrays. `.to contain exactly()` sounds like it will solve this problem going forward. I have some tests in my current project that are solving this problem the same way, so I'll make that change.